terraform/monitoring/dashboards/checkoutservice_dashboard.json

```diff
@@ -0,0 +1,170 @@
+{
+  "displayName": "Checkout Service Dashboard",
+  "gridLayout": {
+    "columns": "2",
+    "widgets": [
+      {
+        "title": "CPU Limit Utilization - Checkout Service",
+        "xyChart": {
+          "dataSets": [
+            {
+              "timeSeriesQuery": {
+                "timeSeriesFilter": {
+                  "filter": "metric.type=\"kubernetes.io/container/cpu/limit_utilization\" resource.type=\"k8s_container\" resource.label.\"cluster_name\"=\"stackdriver-sandbox\" metadata.user_labels.\"app\"=\"checkoutservice\"",
```

Comment:
change stackdriver-sandbox to cloud-ops-sandbox everywhere in this PR. Starting from our next release the GKE cluster will be called cloud-ops-sandbox and filters should reflect that

Reply:
Yep - thanks! made a little oopsie while rebasing. They're all fixed now :) Thanks!